db/migrate/20160515212622_create_join_table_phenotype_snp.rb

```diff
@@ -0,0 +1,17 @@
+class CreateJoinTablePhenotypeSnp < ActiveRecord::Migration
+  def self.up
+    # see rails naming convention for naming JOIN tables
+    create_table :phenotype_snps do |t|
+      t.references :snp
+      t.references :phenotype
+      t.float :score, :default => 0
+      t.timestamps
+    end
+
+    add_index :phenotype_snps, [:snp_id, :phenotype_id], :unique => true
+  end
+
+  def self.down
+    drop_table :phenotype_snps
+  end
```

Comment:
If you rename `self.up` to `change` and remove the `self.down` method, Rails can infer the corresponding `drop_table` from the `create_table` call.